### DISCUSSION TITLE
Is channel width set to fix 20MHz?

### DISCUSSION THREAD

**Discussion by CableMate** [nzymedefense/nzyme#1016] [Questions]
The command iw dev <interface name> info shows me, that my interface is set to 20MHz indpendent of the channel and band.

Does it make sense to make the Bandwidth configurable? 
My AWUS036ACM is able to use HT20|HT40+|HT40-|80MHz|160MHz.

**Comment by lennartkoopmann** [comment]
Yes, there will be a future options to cycle through different channel widths as well.

**Comment by CableMate** [comment]
Perfect!
